CHANGELOG.md

```diff
@@ -0,0 +1,5 @@
+# Changelog Short Message Service
+
+# v0.1.0 - Initial version
+
+**Initial version of SMS.YAML and sms-delivery-notification-subscription.yaml.**
```

Comment:
An "Initial" version within the [API Versioning](https://wiki.camaraproject.org/x/a4BaAQ) nomenclature will be later public-release versions 0.1.0 (after alpha and rc phase). Here you have the first alpha versions of that.
```suggestion
** This release contains the first alpha versions of SMS.YAML 0.1.0 and sms-delivery-notification-subscription.yaml 0.1.0**
```